packages/ts-client/src/webrtc/transceivers.ts

```diff
@@ -1,49 +1,59 @@
 import { EndpointWithTrackContext, TrackContextImpl } from "./internal";
 import { RemoteTrackId } from "./types";
 
+type Mid = string
+type TrackId = string;
+type MidToTrackId = Record<Mid, TrackId>
+
 export const getMidToTrackId = <EndpointMetadata, TrackMetadata>(
   connection: RTCPeerConnection | undefined,
   localTrackIdToTrack: Map<RemoteTrackId, TrackContextImpl<EndpointMetadata, TrackMetadata>>,
   midToTrackId: Map<string, string> = new Map(),
   localEndpoint: EndpointWithTrackContext<EndpointMetadata, TrackMetadata>,
-): Record<string, string> | null => {
+): MidToTrackId | null => {
   if (!connection) return null;
 
-  const active = getActive(connection, localTrackIdToTrack)
-  const muted = getMuted(midToTrackId, localEndpoint)
+  // - negotiated unmuted tracks: tracks added in previous negotiation, data is being transmitted
+  // - not yet negotiated tracks: tracks added in this negotiation, data will be transmitted after successful negotiation
+  const transceiver = getTransceiverMapping(connection, localTrackIdToTrack)
+
+  // - negotiated unmuted tracks: tracks added in previous negotiation, data is being transmitted
```

Comment:
Why do we have the same comment two times?

Reply:
Because it's an intersection of these two functions